examples/code/fcm/query_handler.mlt

```diff
@@ -52,7 +52,7 @@ val sexp_of_u : u -> Sexp.t = <fun>
 |}];;
 sexp_of_u {a=3;b=7.};;
 [%%expect ocaml {|- : Sexp.t = ((a 3) (b 7))|}];;
-u_of_sexp (Sexp.of_string "((a 43) (b 3.4))");;
+u_of_sexp (Core_kernel.Sexp.of_string "((a 43) (b 3.4))");;
```

Comment:
Why is Core_kernel still needed here?  I thought Sexp was part of Base...

Reply:
Base has s-expressions, but no printers or parsers. Those are in Parsexp and Sexplib. Core_kernel does include those. Maybe we should reference Parsexp directly, but the API is a little more complicated.